src/ORM/Table.php

```diff
@@ -121,6 +121,13 @@ class Table implements RepositoryInterface, EventListenerInterface
     use EventManagerTrait;
 
     /**
+     * Name of default validation set.
+     *
+     * @var string
+     */
+    const VALIDATOR_DEFAULT = 'default';
```

Comment:
Funny, I initially had that before looking at the [other constants][1].

Many of them seem to be written in this style, usually when multiple constants are involved.

Looking again though, I also see constants like `DEFAULT_HOST` and `DEFAULT_PORT`.

Happy to change it to whatever works best.

[1]: https://gist.github.com/deizel/9230ab493ada786358bf